src/custom/pages/Swap/index.tsx

```diff
@@ -124,6 +126,67 @@ const SwapModWrapper = styled(SwapMod)`
   }
 `
 
+const fadeIn = `
+  @keyframes fadeIn {
+    0% {
+      opacity: 0;
+    }
+    100% {
+      opacity: 1;
+    }
+  }
+`
+
+interface LoadingCowImgProps {
+  maxWidth?: string
+  showLoader: boolean
+}
+
+const LoadingCowImg = styled.img<LoadingCowImgProps>`
+  max-width: ${({ maxWidth = '60px' }) => maxWidth};
+  opacity: ${({ showLoader }) => (showLoader ? 1 : 0)};
+
+  transition: opacity 0.4s ease-in-out;
```

Comment:
I want make comments on the styling, I know this is being reiterated in other PRs, where we will think the best place and way to represent it.

Reply:
@biocom will address it later